tests/test_lmp.py

```diff
@@ -79,23 +80,43 @@ def test_latest(self):
             self.assertEqual(dp['market'], self.MARKET_CHOICES.fivemin)
             self.assertEqual(dp['freq'], self.FREQUENCY_CHOICES.fivemin)
 
-    def test_date_range(self):
+    def date_range(self, market):
```

Comment:
We don't have a firm convention for this, but I usually do something more like `run_xx_test`